x-pack/solutions/security/plugins/security_solution/public/sourcerer/containers/use_init_sourcerer.tsx

```diff
@@ -57,7 +57,7 @@ export const useInitSourcerer = (
   }, [addWarning, defaultDataView.error]);
 
   const getTimelineSelector = useMemo(() => timelineSelectors.getTimelineByIdSelector(), []);
-  const activeTimeline = useDeepEqualSelector((state) =>
+  const { savedObjectId: timelineSavedObjectId } = useDeepEqualSelector((state) =>
```

Comment:
There was no need to rely on the full timeline object, when all we need to reference is the `savedObjectId`. `useEffects` were unnecessarily re-run when the timeline object reference changed